### ISSUE TITLE
Background only apps are not shown in app restriction list

### ISSUE THREAD

**Issue by naveenmah** [googlesamples/android-testdpc#3]
I am developing a [PrintService](https://developer.android.com/reference/android/printservice/PrintService.html) which is just a background service with no activities. 

I would like to add restrictions to this printservice eg. print only in mono, always print in duplex. etc.
The issue is when testing restrictions with TestDpc app, my print service is not shown in the list. 

If I add a dummy activity with action main then it works fine and shows restrictions also.

Is this as per design? Can background only apps have restrictions? or is this just a issue of TestDPC app?

**Comment by hex539** [comment]
Looks like a bug in TestDPC, it's perfectly fine for background services to support restrictions.

Caused by a check for whether the app is launchable, in [ManageAppFragment](https://github.com/googlesamples/android-testdpc/blob/master/app/src/main/java/com/afwsamples/testdpc/common/ManageAppFragment.java#L91-93):

```
    if ((mPackageManager.getLaunchIntentForPackage(applicationInfo.packageName)) != null) {
        launchableAppList.add(applicationInfo);
    }
```

For now, as a workaround, you could just comment that check out.

**Comment by naveenmah** [comment]
Thanks!!

**Comment by ming030890** [comment]
Addressed in 3.0.3
